tools/builds/make.bat

```diff
@@ -1 +1,5 @@
-cmake -G "Visual Studio 12" -D CMAKE_INSTALL_PREFIX=install -D CMAKE_BUILD_TYPE=Release "--build build" .
+set BUILDDIR=%1
+IF [%BUILDDIR%] == [] (
+    set BUILDDIR=.
+)
```

Comment:
BUILDDIR has now been replaced with SRCDIR to better convey what it's used for :-)